src/PointwiseFunctions/GeneralRelativity/Python/Bindings.cpp

```diff
@@ -63,6 +64,13 @@ void bind_spacetime_impl(py::module& m) {  // NOLINT
 
 template <size_t Dim>
 void bind_impl(py::module& m) {  // NOLINT
+  m.def(
+      "derivative_inverse_spatial_metric",
```

Comment:
Please name this exactly like the C++ function: `deriv_inverse_spatial_metric` (it's easier to work with the pybindings if they mirror the C++ interface exactly)